source/system-administrators/activities/reindexing-content.rst

```diff
@@ -39,23 +46,40 @@ Clear index through API
 .. WARNING::
   This action will delete all content matching the query, review carefully the Solr index & the site name before executing the command.
 
+After sending the CURL command, you will then get a response like this:
+
+.. code-block:: xml
+
+   <?xml version="1.0" encoding="UTF-8"?>
+   <response>
+      <lst name="responseHeader"><int name="status">0</int><int name="QTime">1690</int></lst>
+   </response>
+
+|
+
 ^^^^^^^^^^^^^^^^^^
 Delete index files
 ^^^^^^^^^^^^^^^^^^
 
+To delete any existing content in the index by deleting the index files, do the following:
+
 #. Make sure Tomcat and Solr have been stopped.
 #. Inside your Crafter installation, go to ``data/indexes/{siteName}``.
```

Comment:
Is this line necessary? Seems like the next already indicates the path to delete.